stonks/__init__.py

```diff
@@ -0,0 +1,64 @@
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+from webdriver_manager.firefox import GeckoDriverManager
+from bs4 import BeautifulSoup
+import time
+import re
+from .types import StonksResult
+
+
+class Handler:
+    """
+    Insert Your FII code to get the information about it
+     INPUT
+     - Ex: HGLG11
+
+     OUTPUT
+       {price: '00,00', div_yield: '0,00%', equity_value: '00,00', name:'HGLG11' }
+    """
+    base_url = 'https://www.fundsexplorer.com.br/funds'
+    beautiful_soup = BeautifulSoup()
+
+    def __init__(self):
+        options = Options()
+        options.headless = True
+        self.driver = webdriver.Firefox(options=options, executable_path=GeckoDriverManager().install())
+
+    def navigate(self):
+        self.driver.get(self.base_url)
+        time.sleep(2)
+
+    def insert_value(self, symbol: str):
+        self.driver.get(f'{self.base_url}/{symbol.upper()}')
+
+    def get_page(self):
+        source_page = self.driver.page_source
+        return BeautifulSoup(source_page, 'html.parser')
```

Comment:
It's cool to have one space before the return statement—this help to read and find the returning clauses.